go/tools/builders/env.go

```diff
@@ -157,13 +157,15 @@ func absEnv(envNameList []string, argList []string) error {
 }
 
 func runAndLogCommand(cmd *exec.Cmd, verbose bool) error {
+	var buffer bytes.Buffer
+	formatCommand(&buffer, cmd)
```

Comment:
Nit: I think this would be more readable if `formatBuffer` just returned a `string`. There's probably no good reason for it to accept an `io.Writer` anyway.